### PR TITLE
Admin UI revamped + Discord notification (right from admin panel)

### LINKED ISSUE #1164: bug: admin dashboard unable to change urls
Topbar hides the elements below
Screenshot 2024-09-09 at 8 31 26 PM

### PR DESCRIPTION
**Pull request by amanbairagi30**
### PR Fixes:
- 1 Admin UI revamped
- 2 Discord notification (right from admin panel)

Resolves #1164 

#Discord Notification: 

https://github.com/user-attachments/assets/e165b98b-8f3c-4205-b9a6-631ca5369a70

image
image
image
image
image

### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] I assure there is no similar/duplicate pull request regarding same issue

### CODE REVIEW

**Comment by amanbairagi30** [comment]
@hkirat @devsargam

**Comment by devsargam** [comment]
Will check within this week. Thanks :)

**Review by devsargam — APPROVED** [review]

**Comment by devsargam** [comment]
Checking locally

File: src/app/api/admin/content/route.ts [hunk 1]
```diff
@@ -1,6 +1,40 @@
...
+  title: string;
+  courseTitle: string;
+  courseId: number;
+  currFolderId: number;
+  mediaId: number;
+}
+
+const sendUpdateToDiscord = async (data: DiscordData) => {
+  const body = {
+    content: 'Hello @everyone',
```
Comment (devsargam) [inline src/app/api/admin/content/route.ts:17]: Should not tag everyone

Comment (devsargam) [inline src/app/api/admin/content/route.ts:17]:
```suggestion
    content: 'Hello everyone',
```

**Review by devsargam — CHANGES_REQUESTED** [review]

**Comment by devsargam** [comment]
Thank u sir @amanbairagi30

**Pull request merged**